example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -525,12 +586,13 @@ describe('The S3 Ingest Granules workflow', () => {
         s3BrowseImageUrl = getDistributionFileUrl({ bucket: files[2].bucket, key: files[2].filepath, urlType: 's3' });
         s3CredsUrl = resolve(process.env.DISTRIBUTION_ENDPOINT, 's3credentials');
       } catch (error) {
-        beforeAllError = error;
+        subTestSetupError = error;
       }
     });
 
     beforeEach(() => {
       if (beforeAllError) fail(beforeAllError);
+      if (subTestSetupError) fail(subTestSetupError);
     });
 
     it('publishes the granule metadata to CMR', async () => {
```

Comment:
Do you omit this 
```
      if (beforeAllError || subTestSetupError) throw SetupError;
``` 
for the first test of some describe blocks because it will be covered in the `beforeEach`?